pyproject.toml

```
@@ -41,19 +41,21 @@ watchdog = { version = "^0.9.0", optional = true }
 GitPython = { version = "^3.0", optional = true }
 docker = { version = "^4.2.0", optional = true }
 pymongo = { version = "^3.11.2", optional = true }
+etcd3 = { version = "^0.12.0", optional = true }
 
 [tool.poetry.extras]
 # core extras
 dill = ["dill"]
 whoosh = ["Whoosh"]
 mongo = ["pymongo"]
+etcd3 = ["etcd3"]
```

Extra name can be simpler as:

```suggestion
etcd = ["etcd3"]
```